bundles/org.openhab.ui/web/src/assets/definitions/blockly/ohblocks_ephemeris.js

```diff
@@ -0,0 +1,128 @@
+/*
+* Ephemeris provides calendar related information
+* This code has been originally provided by https://github.com/bigbasec
+* @author stefan.hoehn
+*
+* See more background info on openHAB ephemeris here: https://www.openhab.org/docs/configuration/actions.html#ephemeris
+*/
+import Blockly from 'blockly'
+import { FieldItemModelPicker } from './ohitemfield'
+
+export default function defineOHBlocks_Ephemeris (f7) {
+  Blockly.Blocks['oh_ephemeris_basic'] = {
+    init: function () {
+      this.appendDummyInput()
+        .appendField(new Blockly.FieldDropdown([['isWeekend', 'isWeekend'], ['getBankHolidayName', 'getBankHolidayName'], ['getNextBankHoliday', 'getNextBankHoliday'], ['isBankHoliday', 'isBankHoliday']]), 'type')
```

Comment:
These should have proper natural language labels:
- it is the weekend
- it is a bank holiday

I don't understand what this block is for, depending on what's selected in the dropdown, it retrieves a String value a boolean. It's clearly not the right way of designing blocks.
Probably best to add individual blocks like below (which I don't see in the toolbox)